src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -232,9 +232,11 @@ const ApplicationDetails: React.FC = () => {
       }));
 
       setDocuments(documents);
-      if (documents.length > 0 && documents[0].status) {
-        setIsVerifyButtonVisible(false);
-      }
+
+      const anyDocumentNotVerified =
+        documents.length > 0 && documents.some((doc: any) => doc.status == null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)